Source/CenterCellCollectionViewFlowLayout.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+class CenterCellCollectionViewFlowLayout: UICollectionViewFlowLayout {
+  
+  override func targetContentOffsetForProposedContentOffset(proposedContentOffset: CGPoint, withScrollingVelocity velocity: CGPoint) -> CGPoint {
+    
+    if let collectionView = self.collectionView {
+      
+      let collectionViewBounds = collectionView.bounds
+      let halfWidth = collectionViewBounds.size.width * 0.5;
```

Comment:
@vadymmarkov wouldn't it make more sense to use division here instead of multiplication ?
Got confused when I glanced at the object name contra the math. What do you think?